DEVELOPER_GUIDE.md

```diff
@@ -230,6 +230,11 @@ $ wsl -d docker-desktop
 $ sysctl -w vm.max_map_count=262144
 ```
 
+#### Debugging
+You can debug the OpenSearch Dashboards server by
+1. Attaching your debugger client to the debug port on port `9229`. If you're using VSCode, you can use the `attach-to-server` debug configuration provided.
```

Comment:
No love for webstorm? :(

Reply:
I'd love this for webstorm too! Do you know if webstorm has a similar debugging file? We can include as followup.